src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -73,7 +73,6 @@ export const _UserAvatar: React.FC<UserAvatarProps> = ({
   const statusIndicatorClasses = classNames("UserAvatar__status-indicator", {
     "UserAvatar__status-indicator--online": isOnline,
     [`UserAvatar__status-indicator--${status}`]: isOnline && status,
-    [`UserAvatar__status-indicator--${size}`]: size,
```

Comment:
NOTE: not necessary anymore. It's more robust to put the size modifier at the top component level and have all descendants react accordingly.